riff-raff/app/views/continuousDeployment/list.scala.html

```diff
@@ -36,7 +36,7 @@ <h4 class="alert-heading">Change Freeze</h4>
                 <tr @if(config.trigger == Disabled){ class="danger" }>
                     <td>@utils.DateFormats.Short.print(config.lastEdited) by <span class="label label-default">@config.user</span></td>
                     <td>@config.projectName</td>
-                    <td>@config.branchRE.toString()</td>
+                    <td>@config.branchMatcher.getOrElse("*")</td>
```

Comment:
`*` as a default feels a bit risky; presumably it means _any_ branch will be continuously deployed?

Reply:
But yes I agree that this seems risky! Perhaps in a future PR this could become a required field?